data-store/chapter.md

```diff
@@ -182,9 +129,29 @@ from the logical structure of the data
 from the contents of the key/value store
 (the association of key "a" to value "foo"; the public API).
 
+[TODO: Sentence or two about the common pattern of separating logical from
+physical, and why it's used in many database designs.]
```

Comment:
"Many databases separate the logical and physical aspects as it is is often useful to provide alternative implementations of each to get different performance characteristics, e.g. DB2's SMS (files in a filesystem) vs DMS (raw block device) tablespaces, or MySQL's [alternative engine implementations](http://dev.mysql.com/doc/refman/5.7/en/storage-engines.html)."